Python/compile.c

```diff
@@ -8393,10 +8426,10 @@ dump_instr(struct instr *i)
 
     *arg = '\0';
     if (HAS_ARG(i->i_opcode)) {
-        sprintf(arg, "arg: %d ", i->i_oparg);
+        sprintf(arg, "arg: %d ", OPARG_VALUE(i->i_oparg));
     }
     if (HAS_TARGET(i->i_opcode)) {
-        sprintf(arg, "target: %p [%d] ", i->i_target, i->i_oparg);
```

Comment:
since we have the type information and it may be relevant to debugging, perhaps the debug dump should include it?